### PR TITLE
Feature: ScreenWipe

### PR DESCRIPTION
**Pull request by HIHIQY1**
Is already present in level_0_screenwipe, when entering / exiting doors. Adding the ScreenWiper prefab to a scene adds this functionality. It is already integrated with the doors themselves.

https://github.com/smooth-soup-studios/oh-my-cult/assets/26181740/7929180c-e994-4140-8844-9682e72eb6b3

### CODE REVIEW

File: Assets/Scripts/Controllers/Objects/DoorController.cs [hunk 1]
```diff
@@ -43,13 +44,25 @@ public class DoorController : MonoBehaviour {
...
 			_isTransporting = true;
 
-			GameManager.Instance.LoadScene(
-				TransportTo switch {
-					TransportDestination.House => "houses",
-					TransportDestination.Special => "boss_arenas",
-					_ => "level_0"
-				}
-			);
+			Debug.Log("dd, YAEH");
```
Comment (Justwouter) [inline Assets/Scripts/Controllers/Objects/DoorController.cs:47]: dd indeed.
There seems to be a bug or visual glitch when exiting a house whilst holding one of the movement keys.
The wipe seems to do a double take or smth.

Comment (HIHIQY1) [reply Assets/Scripts/Controllers/Objects/DoorController.cs:47]: This is, however, not my issue, so I suppose just removing the debug log and the PR is all good-to-go?

Comment (Justwouter) [reply Assets/Scripts/Controllers/Objects/DoorController.cs]: I'd argue that is is in fact your issue. While it is not necessarily directly caused by the screenwipe code, it is caused by the implementation of it in the doorcontroller and will need to be fixed anyways so why not now?

Got annoyed and did it myself this time.

**Review by Justwouter — APPROVED** [review]
Lgtm

**Pull request merged**